operator/cmd/mesh/profile-common.go

```diff
@@ -187,6 +199,15 @@ func unmarshalAndValidateIOP(crYAML string, force bool) (*v1alpha1.IstioOperator
 	return iops, iopsYAML, nil
 }
 
+func translateICPToIOP(icp string) (string, error) {
+	translateConfigPath := "translateConfig/translate-ICP-IOP-1.5.yaml"
```

Comment:
translateConfig path should be const somewhere, as well as filename prefix. 
1.5 version should be passed in from version. OperatorBinaryVersion.

Reply:
Fixed.